src/test/java/de/tum/in/www1/artemis/domain/GuidedTourSettingsTest.java

```diff
@@ -0,0 +1,24 @@
+package de.tum.in.www1.artemis.domain;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class GuidedTourSettingsTest {
+
+    @Test
+    public void createFromJson() {
+        String guidedTourSettingsJson = "{\n" + "\t\"showCourseOverviewTour\" : true,\n" + "\t\"showNavigationTour\" : false,\n" + "\t\"showProgrammingExerciseTour\" : true,\n"
```

Comment:
I would use `Json.createObjectBuilder()` instead: https://docs.oracle.com/javaee/7/api/javax/json/JsonObjectBuilder.html

Reply:
@elkbreeder I think for this test it's simpler to just use the string